x-pack/plugin/sql/qa/src/main/resources/agg.sql-spec

```diff
@@ -279,6 +279,8 @@ aggMinWithAlias
 SELECT gender g, MIN(emp_no) m FROM "test_emp" GROUP BY g ORDER BY gender;
 aggMinOnDateTime
 SELECT gender, MIN(birth_date) m FROM "test_emp" GROUP BY gender ORDER BY gender;
```

Comment:
If this test here wasn't complaining before, this means that our SQL spec test case infrastructure isn't actually comparing an actual date type of data, right? Otherwise, the test would have failed.
If so, I would like to see a test with `MIN(date_field)` in CSV spec test case infra, if possible. I have seen the unit test that compares the metric agg key with an actual date (`assertEquals(DateUtils.asDateTime((long) innerValue , zoneId), extractor.extract(bucket));`) but I believe our tests would be more complete if there would be an integration test as well (CSV-spec in this case).